.github/workflows/cd.yml

```diff
@@ -83,54 +83,6 @@ jobs:
           draft: ${{ env.ACTIONS_STEP_DEBUG == 'true' }}
           prerelease: false
           files: ${{ steps.sign.outputs.asset_path }}
-  deploy-macos-bundle:
-    needs: deploy
-    env:
-      B2_OSX_CODE_SIGNING_CERTIFICATE: ${{ secrets.B2_OSX_CODE_SIGNING_CERTIFICATE }}
-      B2_OSX_CODE_SIGNING_CERTIFICATE_NAME: ${{ secrets.B2_OSX_CODE_SIGNING_CERTIFICATE_NAME }}
-      B2_OSX_CODE_SIGNING_CERTIFICATE_PASSWORD: ${{ secrets.B2_OSX_CODE_SIGNING_CERTIFICATE_PASSWORD }}
-      B2_OSX_NOTARY_NAME: ${{ secrets.B2_OSX_NOTARY_NAME }}
-      B2_OSX_NOTARY_PASSWORD: ${{ secrets.B2_OSX_NOTARY_PASSWORD }}
-    runs-on: macos-10.15
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Set up Python ${{ env.PYTHON_DEFAULT_VERSION }}
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ env.PYTHON_DEFAULT_VERSION }}
-      - name: Install dependencies
-        run: python -m pip install --upgrade nox pip setuptools
-      - name: Bundle the distribution
-        id: bundle
-        run: nox -vs bundle >> $GITHUB_OUTPUT
-      - name: Import certificate
-        uses: apple-actions/import-codesign-certs@v1
-        with:
-          p12-file-base64: ${{ env.B2_OSX_CODE_SIGNING_CERTIFICATE }}
-          p12-password: ${{ env.B2_OSX_CODE_SIGNING_CERTIFICATE_PASSWORD }}
-      - name: Sign the bundle
-        id: sign
-        run: nox -vs sign -- '${{ env.B2_OSX_CODE_SIGNING_CERTIFICATE_NAME }}' >> $GITHUB_OUTPUT
```

Comment:
This means that we can remove a function that performs a signing for macOS from the `noxfile.py`.

Reply:
I'm a bit afraid that we'll need it back one day and a bit hopeful that we'll one day have "pyinstaller but better", but you are right, it should be removed